cpp/dolfinx/io/cells.cpp

```diff
@@ -341,6 +409,33 @@ std::vector<std::uint8_t> io::cells::perm_gmsh(const mesh::CellType type,
   return io::cells::transpose(map);
 }
 //-----------------------------------------------------------------------------
+std::vector<std::uint8_t> io::cells::perm_discontinuous(mesh::CellType type,
```

Comment:
I think most of the added functions need to be made more expressive.

Reply:
`perm_DG`?